src/main/java/org/apache/commons/text/WordUtils.java

```diff
@@ -738,4 +740,69 @@ private static boolean isDelimiter(final char ch, final char[] delimiters) {
         return false;
     }
 
-}
+    //-----------------------------------------------------------------------
+    /**
+     * Abbreviates a string nicely.
+     *
+     * This method searches for the first space after the lower limit and abbreviates
+     * the String there. It will also append any String passed as a parameter
+     * to the end of the String. The upper limit can be specified to forcibly
+     * abbreviate a String.
+     *
+     * @param str         the string to be abbreviated. If null is passed, null is returned.
+     *                    If the empty String is passed, the empty string is returned.
+     * @param lower       the lower limit.
+     * @param upper       the upper limit; specify -1 if no limit is desired.
+     *                    If the upper limit is lower than the lower limit, it will be
+     *                    adjusted to be the same as the lower limit.
+     * @param appendToEnd String to be appended to the end of the abbreviated string.
+     *                    This is appended ONLY if the string was indeed abbreviated.
+     *                    The append does not count towards the lower or upper limits.
+     * @return the abbreviated String.
+     * @since 2.4
```

Comment:
Sorry I have to correct myself. The tag should be removed as the class already has `@since 1.1`.